.github/workflows/prod-deploy-on-release.yml

```
@@ -0,0 +1,87 @@
+        with:
+          distribution: temurin
+          java-version: 21
+
+      - name: Run tests
+        working-directory: ./server
+        run: |
+          chmod +x ./gradlew
+          ./gradlew test
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)